src/styles/global/_custom-bootstrap.scss

```diff
@@ -1,9 +1,9 @@
 // color overrides
-$primary: $udb-blue;
-$secondary: $udb-grey;
-$success: #5cb85c;
-$danger: #d9534f;
-$light: $white;
+// $primary: $udb-blue;
+// $secondary: $udb-grey;
+// $success: #5cb85c;
+// $danger: #d9534f;
+// $light: $white;
```

Comment:
(The PR is still a draft so you're probably still actively making changes, but just a reminder to not forget to remove these later if we end up not needing them here.)